src/translations/en.json

```
@@ -15,6 +15,7 @@
   "error": "Error:",
   "informational": "Informational:",
   "loading": "Loading",
+  "add": "add",
```

To match `src/translations/en.ts`

```suggestion
  "add": "Add",
```